api/use_cases/get_users_count.py

```diff
@@ -0,0 +1,35 @@
+from api.use_cases import UseCase
+from api.share.requests.users import GetUsersCountRequest
+from api.models.people import PersonalInfo
+from django.db.models.functions import Lower
+
+
+class UsersCount(UseCase):
+
+    def __int__(self):
+        pass
+
+    def process_request(self, request_object: GetUsersCountRequest):
+        # business logic
+        registration_bot_chat_id = request_object.registration_bot_chat_id
+        tg_username = request_object.tg_username
+        phone = request_object.phone
+        email = request_object.email
+        first_name = request_object.first_name
+        last_name = request_object.last_name
+
```

Comment:
utc_timedelta, availability_slots, communication_language_mode, comment

Reply:
What do you mean?

Reply:
I mean that these are new attributes that we now have in `PersonalInfo`

Reply:
Why do they need here?

Reply:
OK, I see your point.